src/tripviewer/web/Models/DataObjects/User.cs

```diff
@@ -5,59 +5,58 @@ namespace Simulator.DataObjects
 
     public partial class User //: BaseDataObject
     {
-        [JsonProperty("id")]
+        [JsonProperty("Id")]
         public Guid Id { get; set; }
 
-        [JsonProperty("firstName")]
+        [JsonProperty("FirstName")]
         public string FirstName { get; set; }
 
-        [JsonProperty("lastName")]
-        [JsonConverter(typeof(ParseStringConverter))]
-        public long LastName { get; set; }
+        [JsonProperty("LastName")]
+        public string LastName { get; set; }
```

Comment:
json serialization error is fixed with this as string and `[JsonConverter(typeof(ParseStringConverter))]` removed. I think we should either remove attribute or change back to lowercase

Reply:
I've removed the `JsonProperty` attributes now. Should be good to go.